### PR TITLE
Add printing option for page orientation

### PR DESCRIPTION
**Pull request by atdotde**
Strangely enough, half of the infrastructure was
already there, it just wasn't hooked up to a UI
element

This was requested on the google group.

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [x] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### CODE REVIEW

File: desktop-widgets/printoptions.cpp [hunk 1]
```diff
@@ -47,6 +48,7 @@ void PrintOptions::setup()
 
 	connect(ui.printInColor, SIGNAL(clicked(bool)), this, SLOT(printInColorClicked(bool)));
 	connect(ui.printSelected, SIGNAL(clicked(bool)), this, SLOT(printSelectedClicked(bool)));
+	connect(ui.printLandscape, SIGNAL(clicked(bool)), this, SLOT(printLandscapeClicked(bool)));
```
Comment (bstoeger) [inline desktop-widgets/printoptions.cpp]: Pleased don't add new SIGNAL/SLOT-style connects. In this trivial case you might even add a lambda as in the next line.

File: desktop-widgets/printoptions.ui [hunk 2]
```diff
@@ -93,6 +93,13 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="printLandscape">
+        <property name="text">
+         <string>Page orientation landscape</string>
```
Comment (bstoeger) [inline desktop-widgets/printoptions.ui:99]: This adds a new translation. What's the plan @dirkhh?

File: desktop-widgets/printoptions.cpp (hunk 1 above)
Comment (atdotde) [reply desktop-widgets/printoptions.cpp]: Turned all three into lambdas.

Comment (bstoeger) [reply desktop-widgets/printoptions.cpp]: Thanks!

Comment (dirkhh) [reply desktop-widgets/printoptions.cpp]: yay... more... ƛ - just what I always wanted (just joking... this is the right thing to do here)

File: desktop-widgets/printoptions.ui (hunk 2 above)
Comment (dirkhh) [reply desktop-widgets/printoptions.ui:99]: Clearly I was completely thrown off course and it seems ridiculous to stop everything until I can get my act together.
Admittedly one of the cloud servers going away over night the other week didn't help.

Right now I'm just merging the things that are ready and I'll look into releasing a new version afterwards.

File: desktop-widgets/printoptions.cpp [hunk 3]
```diff
@@ -45,9 +46,16 @@ void PrintOptions::setup()
...
+	connect(ui.printInColor, &QCheckBox::clicked, this, [this](bool value) {
+		printOptions.color_selected = value;
+	});
+	connect(ui.printSelected, &QCheckBox::clicked, this, [this](bool value) {
+		printOptions.print_selected = value;
+	});
+	connect(ui.printLandscape, &QCheckBox::clicked, this, [this](bool value) {
+		printOptions.landscape = value;
+	});
+	connect(ui.resolution, QOverload<int>::of(&QSpinBox::valueChanged), this,  [this](int value) {
```
Comment (dirkhh) [inline desktop-widgets/printoptions.cpp:58]: I will grudgingly admit that this is a perfect use case for lambda functions.

**Review by dirkhh — APPROVED** [review]
looks good to me

**Pull request merged**